modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/legacy_appeals_controller.rb

```diff
@@ -80,4 +81,8 @@ def render_unusable_response_error
     render status: :bad_gateway,
            json: Common::Exceptions::BackendServiceException.new(detail: 'Unusable upstream response')
   end
+
+  def enabled?
+    true if Settings.modules_appeals_api.legacy_appeals
```

Comment:
I don't think `legacy_appeals` as a key denotes whether it's enabled or not - wouldn't `legacy_appeals_enabled: true/false` be more indicative of that fact?

Reply:
OK!